### ISSUE TITLE
how to unpack base file ?

### ISSUE THREAD

**Issue by flayhender** [TekkaGB/AemulusModManager#94]
im pretty sure i did everything right (p5r) but i still got an error when i unpack "value cannot be null" and i got my patch 1cpk and all useu in my folder so idk

**Comment by Fnashmer** [comment]
i got this same problem, after solving the "Not all cpks needed (ALL_USEU.CPK and PATCH1.CPK) are found in top directory" that i fixed just doing "dump romfs" in yuzu to my P5R (for people who got that problem)

**Comment by flayhender** [comment]
yes but thats already what i did and nex ti got this problem right here

**Comment by Radiek** [comment]
I'm having the same issue, would love an answer

**Comment by Fnashmer** [comment]
god just sent a solution right up of us, in resume download Aemulus 6.4.5.7 and do NOT update

**Comment by Fnashmer** [comment]
the original solution is in the issue "Fix the .NET 7 releases (problems with unpacking base files (P5R Switch))"

**Comment by Radiek** [comment]
I did that, and it worked, I unpacked and got some mods but they don't work, am I doing something wrong? What's the output directory?

**Comment by Fnashmer** [comment]
Open Yuzu, right click on your P5R and go to "Open mod folder" or something like that, copy the direction of the folder and paste on output

**Comment by Fnashmer** [comment]
remember to click the hammer to apply mods, and if nothing works try to execute as administrator

**Comment by Radiek** [comment]
I do click on the hammer, and that's what I have my output set to, but the mods still don't work and I tried admin and that didn't work either

**Comment by Fnashmer** [comment]
do you have a up to date Yuzu or a updated P5R? (i don't know if that is but maybe work)

**Comment by Fnashmer** [comment]
or maybe is some mod that is creating the problem (sorry for my english)

**Comment by Radiek** [comment]
I have yuzu updated and the game too, and it can't be a mod conflict because i tried turning off all mods except one and it still wasn't working, and neither were the others one by one

**Comment by flayhender** [comment]
still the same problem for me when it say value cannot be null 
(just in case: output folder is p5r user load 01005CA01580E000    rom path: p5r data games p5r.nsp    emulator path: p5r yuzu.exe)

**Comment by Radiek** [comment]
@flayhender Go to releases and download v6.4.5

**Comment by flayhender** [comment]
ok ty

**Comment by Radiek** [comment]
@flayhender https://github.com/TekkaGB/AemulusModManager/releases/tag/6.4.5

**Comment by Fnashmer** [comment]
if u update download this https://dotnet.microsoft.com/es-es/download/dotnet/7.0

**Comment by Radiek** [comment]
Updating Aemulus didn't work either

**Comment by flayhender** [comment]
i got the unpack done i will see fot he mods now

**Comment by Fnashmer** [comment]
so, i think it's Aemulus who have the problem

**Comment by Radiek** [comment]
Did the mods work for either of you?

**Comment by flayhender** [comment]
well it didnt seemed to work for me

**Comment by flayhender** [comment]
what mod do you use ? i need one where i can directly see if it work or not

**Comment by flayhender** [comment]
where did you go to disable add ons ?

**Comment by Radiek** [comment]
I used the JP font and colorful pack, the colorful pack especially works immediately as you reach the main menu since the characters will have colors and not be grayed out
If you go to yuzu and in the games list you right-click P5R you go to Properties and Add-Ons should be the first tab

**Comment by flayhender** [comment]
hmm weird the only add on i have is called mods and i believe its a folder i created myself

**Comment by flayhender** [comment]
you dl the mod directly in aemulus ?

**Comment by Radiek** [comment]
Yes, did you update Aemulus after unpacking and got the .NET SDK?

**Comment by flayhender** [comment]
i didnt update. where do you got the net sdk ?

**Comment by flayhender** [comment]
oh well i tested the colorful mod and it worked but i try to have the exp multiplier but this one doesnt seem to work tho

**Comment by Fnashmer** [comment]
> Did the mods work for either of you?
i am not in my home rn but i will install mods when i get here

**Comment by flayhender** [comment]
well yes it works for me now ! (i just have to fix my texture problem now lol)

**Comment by Fnashmer** [comment]
what you did to work?

**Comment by Fnashmer** [comment]
nvrm mods work fine

**Comment by Fnashmer** [comment]
![image](https://github.com/TekkaGB/AemulusModManager/assets/108560514/a23cacf2-9dde-4fa2-9942-7f036a20764c)

**Comment by Fnashmer** [comment]
note: u can make mods compatible with Aemulus just drop "BASE" and "EN" (if EN is in the folder obviusly), creating a new package and putting all here, but the PS buttons crashed my game so i desactivate it.
